Data-Visualization-Project.md

```diff
@@ -0,0 +1,83 @@
+# Data Visualization Using D3.js 
+
+### Our goal is to learn the basics of data visualization using D3 through this project. 
+
+![screen shot 2016-05-17 at 5 02 41 pm](https://cloud.githubusercontent.com/assets/8707780/15343407/4622fe20-1c51-11e6-9892-24e8a394b549.png)
+
+
+What is D3.js? 
+
+For those who are not familiar with D3, D3.js is a JavaScript library for manipulating documents based on data. D3 helps you bring data to life using HTML, SVG and CSS. 
+
+D3.js helps you attach your data to DOM (Document Object Model) elements. Then you can use CSS3, HTML, and/or SVG showcase this data. Finally, you can make the data interactive through the use of D3.js data-driven transformations and transitions.
+
+
+## Project Explanation: 
+
+### Scenario : 
+There’s a class of Online students interacting with different topics of the subject and gives quiz on those topics. 
+There are 15 topics and on each topic, we have numbers of student who have taken quiz and have scored in three categories: Low, Mid and High
+
+### For example (Given data ) : 
+Topic:'1’, low:4, mid:13, high:18 
+Topic:’2’,  low:11, mid:12, high:6
+Topic:'3’,low:12, mid:24, high:6 and so on. 
+
+Note that Topic 1 has 4 + 13 + 8 = 35 student has taken the quiz and Topic 2 has 11+12+6 = 29 student and topic 3 has 42 students and so on. 
+
+**We want to make an interactive Bar and Pie charts. For example, a mouse hover on one of the bars will change the pie chart accordingly and vice-versa.  
```

Comment:
~~We want to make an interactive Bar and Pie chart~~ We want to make interactive Bar and Pie charts.